pkg/onchain/website/handler.go

```diff
@@ -61,10 +86,17 @@ func RedirectToDefaultResourceInterceptor(req *interceptor.Interceptor) *interce
 		splited := removeEmptyStrings(strings.Split(req.Request.URL.Path[len(prefix):], "/"))
 
 		if len(splited) == 1 {
+			protocol := "https"
```

Comment:
if the interceptor is moved to the top, this code is useless as the protocole here will be HTTPS when possible.